src/components/TaskCard/TaskCard.tsx

```diff
@@ -24,23 +25,31 @@ export function TaskCard({ task }: Props) {
   return (
     <div>
       <div className="taskCard" onClick={handleEditModal}>
-        <div
-          className="checkbox"
-          data-testid="checkbox"
-          onClick={handleCheckbox}
-        >
-          {!task.isComplete ? (
-            <RadioButtonUncheckedIcon
-              className="ellipse"
-              data-testid="unchecked"
-            />
-          ) : (
-            <CheckCircleIcon className="ellipse" data-testid="checked" />
-          )}
+        <div className="checkboxAndDescription">
+          <div
+            className="checkbox"
+            data-testid="checkbox"
+            onClick={handleCheckbox}
+          >
+            {!task.isComplete ? (
+              <RadioButtonUncheckedIcon
+                className="ellipse"
+                data-testid="unchecked"
+              />
+            ) : (
+              <CheckCircleIcon className="ellipse" data-testid="checked" />
+            )}
+          </div>
+          <p className={`normal ${task.isComplete && "completeDescription"}`}>
```

Comment:
With complex classnames like this you could look into using https://github.com/JedWatson/classnames. It can be helpful when a component needs multiple dynamic classnames.

Reply:
COOL! I love this.